lib/tests/streamlit/elements/markdown_test.py

```diff
@@ -39,6 +42,43 @@ def test_st_markdown(self):
         self.assertEqual(el.markdown.body, "some markdown")
         self.assertEqual(el.markdown.help, "help text")
 
+    def test_works_with_element_replay(self):
```

Comment:
Thanks for adding the new tests 🥳 What do you think about creating a new cached_reply.py file and create paramterized tests there for all widgets similar to how I might do it for the check_fragment_outside_writing tests [here](https://github.com/streamlit/streamlit/pull/8756/files#diff-1732b92fbcf34c3fde20854b7edc951b4f8850df372c6378829fea27462f2a32R299-R323)? Then we would have the cache_element tests at one central points for all elements and could re-use the list of widgets and non-widgets between the different tests.

Reply:
Awesome, thank you! I have just merged [the fragment-PR](https://github.com/streamlit/streamlit/pull/8756), could you please update the `fragment_test` to use the imported lists?

Reply:
Updated 👍

Reply:
Thank you!